cirq-core/cirq/sim/simulator.py

```diff
@@ -157,6 +157,9 @@ def compute_amplitudes(
                 as an integer array where each integer is formed from measured
                 qubit values according to `qubit_order` from most to least
                 significant qubit, i.e. in big-endian ordering.
+                If inputting a binary literal add the prefix 0b or 0B.
+            For example: 0010 can be input as 0b0010, 0B0010, 2, 0x2, etc.
```

Comment:
This needs to be indented by four spaces to align with the line above. Also, you can put "If inputting" on line 159 above that.

(Applies to the comment on lines 221-222 as well.)

Reply:
@viathor Made these changes in the latest commit :)

Reply:
I only meant a part of the "if inputting" line since there is an overall line length limit (and in any case we should stick to the rough right text boundary of the surrounding text). I'm approving this PR, but note that you will need to take care of this before the merge since at the moment the line-too-long issue makes the formatter check go red.

Reply:
Thanks! The last two recent commits resolve this (only 'if inputting' is on the previous line).